frontend/src/main/pages/EnrollmentData/EnrollmentDataGraphPage.js

```diff
@@ -0,0 +1,15 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+// Stryker disable all : placeholder for future implementation
+export default function EnrollmentDataGraphPage() {
+  return (
+    <BasicLayout>
+      <div className="pt-2">
+        <h1>Graph page not yet implemented</h1>
+        <p>
+          <a href="/enrollmentdata/search">Search</a>
+        </p>
+      </div>
+    </BasicLayout>
+  );
+}
```

Comment:
Delete this; you don't need it and aren't using it.

The new EnrollmentHistoryGraph woudl be displayed, when it is finished and working, on this page: `frontend/src/main/pages/CourseDetails/CourseDetailsIndexPage.js`, which already exists.